enable/savage/trait_defs/ui/qt4/svg_button_editor.py

```diff
@@ -93,7 +93,7 @@ def prepare(self, parent):
         """
         name = self.extended_name
         if name != "None":
-            self.context_object.on_trait_change(
+            self.context_object.observe(
```

Comment:
🚨 This depends on changes to be made in traitsUI ! 🚨 

similarly the changes made in `enable/savage/trait_defs/ui/wx/svg_button_editor.py`

I'm thinking we probably want to undo these changes here and hold off until a traitsUI release?

An alternative is to overwrite the `_update_editor` method here on the subclass and change the signature.  However, even with that, traitsUI is still responsible for removing this listener in the base Editor classes `dispose` method.  We could also overwrite that here, but at that point things are getting a little ugly and its probably better to just hold offf?

Reply:
I've left a comment on the issue: https://github.com/enthought/enable/issues/416#issuecomment-794107671

Reply:
OK. I think we're good here